tests/Language/VisitorTest.php

```diff
@@ -187,9 +187,7 @@ public function testAllowsEditingNodeOnEnterAndOnLeave() : void
                         $selectionSet = $node->selectionSet;
 
                         $newNode               = clone $node;
-                        $newNode->selectionSet = new SelectionSetNode([
-                            'selections' => [],
```

Comment:
This empty array assignment was a type violation. PHPStan didn't catch it because we haven't annotated our config objects.

Reply:
Tightening the types in `Printer` bubbled up a bunch more of those violations.